### ISSUE TITLE
Add option to toggle on hold/release.

### ISSUE THREAD

**Issue by phuein** [Anc813/MicMute#19]
Hi. Just started using the app, and it's working great. Would it be possible to add an option that toggles mic muting on key hold and toggles it back on key release?

Would be extra cool if it detected when the toggle key is being held for over 500ms and triggers this behavior automatically.

**Comment by Anc813** [comment]
@phuein it similar to push to talk feature  https://github.com/Anc813/MicMute/issues/9
Unfortunately, this project uses Shortcut library https://github.com/AlexArchive/Shortcut
And it can only notify about shortcut press event, not keyup or keydown (keyup and keydown are required to capture hold/release events)

**Comment by phuein** [comment]
Yeah, the Shortcut library was last updated on Dec 19, 2014, so not much to work with there. I implemented this with a workaround. Far from perfect, but I didn't see myself sending a PR to a much older project.
